docs/buildkitd.toml.md

```diff
@@ -0,0 +1,71 @@
+# /etc/buildkit/buildkitd.toml
+
+## NAME
+
+buildkitd.toml - configuration file for buildkitd
+
+
+## DESCRIPTION
+
+The TOML file used to configure the buildkitd daemon settings has a short
+list of global settings followed by a series of sections for specific areas
+of daemon configuration.
+
+## EXAMPLE
+
+The following is a complete **buildkitd.toml** default configuration example,
+please note some of configuration is only good for edge cases or incomplete
+implemented such as **UserRemapUnsupported**, please take care of it carefully.
+
+```
+debug = true
+root = "/var/lib/buildkit"
+insecure-entitlements = [ "network.host", "security.insecure" ]
+
+[grpc]
+  address = [ "unix:///run/buildkit/buildkitd.sock" ]
+  debugAddress = "0.0.0.0:6060"
+  uid = 0
+  gid = 0
+  [grpc.tls]
+    cert = "/etc/buildkit/tls.crt"
+    key = "/etc/buildkit/tls.key"
+    ca = "/etc/buildkit/tlsca.crt"
+
+[worker.oci]
+  enabled = true
+  platforms = [ "linux/amd64", "linux/arm64" ]
+  snapshotter = "auto"
+  rootless = false
+  noProcessSandbox = false
```

Comment:
`run subprocesses in main pid namespace. useful for running rootless buildkit inside a container`